flake.nix

```diff
@@ -0,0 +1,62 @@
+{
+  # example taken from https://github.com/tomhoule/rust-nix-wasm32-unknown-unknown-example/blob/main/flake.nix
+  description = "Minimal rust wasm32-unknown-unknown example";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    rust-overlay = {
+      url = "github:oxalica/rust-overlay";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, rust-overlay }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        overlays = [ rust-overlay.overlays.default ];
+        pkgs = import nixpkgs { inherit system overlays; };
+        rust = pkgs.rust-bin.fromRustupToolchainFile ./volvelle-wasm/rust-toolchain.toml;
+        inputs = [ rust pkgs.wasm-bindgen-cli ];
+      in
+      {
+        packages.codex32-website = pkgs.rustPlatform.buildRustPackage {
+          pname = "codex32-website";
+          version = "1.0.0";
+
+          src = ./volvelle-wasm/.;
+
+          cargoLock = {
+            lockFile = ./volvelle-wasm/Cargo.lock;
+          };
+
+          nativeBuildInputs = inputs;
+
+          buildPhase = ''
+            echo 'Creating out dir...'
+            mkdir -p $out/target;
+            mkdir -p $out/www/pkg;
+            
```

Comment:
In 19b21daba66ba1a980329771f2a53681f1a54fff:

trailing whitespace

Reply:
I think the commit you reference is no longer and has been replaced by 87f73d